tworld.c

```diff
@@ -1547,6 +1547,9 @@ static void findlevelfromhistory(gamespec *gs, char const *name)
     }
 }
 
+#define OPEN_PAREN (
+#define CLOSED_PAREN )
```

Comment:
These don't appear to be used...

Reply:
Used to be for `PRODUCE_SINGLE_COLUMN_TABLE` to have function calls in `L`/`R`, but I changed the code to not do that, and forgot the parenthesis.